src/test/resources/output/json/00219/HT002192569.json

```diff
@@ -0,0 +1,172 @@
+{
```

Comment:
why adding this new test resource - it seems that the already existing resources do already well testing the "related" data.